top_section_styles.css

```diff
@@ -0,0 +1,254 @@
+@import url(fonts.css);
+
+:root {
+    --primary: #3077C6;
+    --primary-dark: #10367C;
+    --secondary-yellow: #F5BD41;
+    --secondary-dark-mode1: #121212;
+    --secondary-dark-mode2: #1D1E1F;
+    --secondary-dark-mode3: #4D4D4D;
+    --primary-stroke: #F3F3F4;
+    --primary-text: #383838;
+    --primary-white: #FFFFFF;
+    --secondary-text: #BFBFBF;
+    --primary-bg: #EAF0F9;
+}
+
+body {
+    margin: 0;
+}
+.big-container {
+    width: 1440px;
+    height: 900px;
+    min-height: 100vh;
+    background-image: url("images/booking_images/kastelmeccano.jpg");
+    background-repeat: no-repeat;
+    background-position: top;
+
+}
+
+.big-upper {
+    display: flex;
+    justify-content: space-between;
+    margin: 50px 100px 100px 0;
+    align-items: center;
+}
+
+.upper {
+    display: flex;
+    justify-content: space-between;
+    width: 345px;
+    align-items: center;
+    margin-top: 50px;
+}
+
+.words {
+    display: flex;
+    justify-content: space-between;
+    /*margin-top: 50px;*/
+    width: 200px;
+    align-items: center;
+}
+
+.icons {
+    display: flex;
+    justify-content: space-between;
+    /*margin-top: 50px;*/
+    width: 95px;
+    align-items: center;
+}
+
+.image, .svg {
+    width: 205px;
+    height: 40px;
+    fill: var(--primary);
+    margin-top: 50px;
+    margin-left: 100px;
+    align-items: center;
+}
+
+.stays {
+    width: 60px;
+    height: 30px;
+    color: var(--primary-white);
+    font: 24px Roboto, sans-serif;
+    font-weight: 400;
+    line-height: 28px;
+    align-items: center;
+}
+
+.attractions {
+    width: 120px;
+    height: 30px;
+    color: var(--primary-white);
+    font: 24px Roboto, sans-serif;
+    font-weight: 400;
+    line-height: 28px;
+    align-items: center;
+}
```

Comment:
as I have already sad it's better to use a common variant to set simple properties to different classes